app/scripts/controller-init/messengers/index.ts

```diff
@@ -6,6 +6,9 @@ import {
 import {
   getCronjobControllerMessenger,
   getExecutionServiceMessenger,
+  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
```

Comment:
Interesting, what is the motivation behind adding fencing?

Presumably this is only used by keyring snaps, but ideally we'd minimize code fences where they're not especially beneficial (e.g. to reduce bundle size, or to avoid some code that _won't work_ at all outside the fenced condition).

Reply:
Ah interesting. OK, fair enough, this makes sense then. We can consider getting rid of the fence separately I think.